cookbook-react/src/recipes/screens/login/index.test.tsx

```diff
@@ -104,9 +102,10 @@ describe('#Login', () => {
         await fireEvent.submit(form);
       });
 
-      expect(mockLogin).toHaveBeenCalled();
       expect(mockSetStateUser).toHaveBeenCalled();
+      expect(mockSetStateUser).toHaveBeenCalledWith({ sessionToken: 'token', id: 1234 });
```

Comment:
Is it really necessary to test if it was called AND it was called with params? I think the `toHaveBeenCalledWith` is enough 🤔